akka-http-core/src/test/scala/akka/http/impl/engine/client/HighLevelOutgoingConnectionSpec.scala

```diff
@@ -91,5 +91,6 @@ class HighLevelOutgoingConnectionSpec extends AkkaSpec with ScalaFutures {
       a[One2OneBidiFlow.OutputTruncationException.type] should be thrownBy Await.result(x, 3.second)
       binding.futureValue.unbind()
     }
+
```

Comment:
I removed the test, hitting random website in tests does not seem like a good idea.